templates/data/istio-module-data.yaml

```diff
@@ -229,3 +229,39 @@ templates:
                     - vm_job_make
                     - vm_job_k3d
                     - vm_job_labels
+              - jobConfig:
+                  name: nightly-istio-performance-test-gcp
```

Comment:
As far as I know we also want to evaluate Istio performance on Istio version bump. Should we define a job that runs on a specific branch naming convention or is a documentation how to execute this tests in case of a bump enough, since we'd only need to create the cluster and can then re-use the make target used in this job?
What's your opinion?